src/main/java/managers/PageObjectManager.java

```diff
@@ -1,18 +1,34 @@
 package managers;
 
+import base.BasePage;
+import exceptions.PageNotFoundException;
 import org.openqa.selenium.WebDriver;
 import pages.*;
 
 public class PageObjectManager {
     private WebDriver driver;
     private HomePage homePage;
+
     public PageObjectManager(WebDriver driver) {
         this.driver = driver;
     }
 
-    public HomePage getHomePage () {
-      return (homePage == null) ? homePage = new HomePage(this.driver) : homePage;
+    public HomePage getHomePage() {
+        return (homePage == null) ? homePage = new HomePage(this.driver) : homePage;
```

Comment:
to much space

Reply:
I am not sure what you mean :) That additional space before `return` was generated automatically by InteliJ formatting function

Reply:
we're using 4 spaces, right? so it's okay I guess